helmfile/charts/gatekeeper-constraints/values.yaml

```
@@ -7,5 +7,11 @@ requireNetworkpolicies:
 requireResourceRequests:
     enable: true
     enforcementAction: dryrun
+restrictTags:
+    enable: true
+    enforcementAction: dryrun
 
 imageRegistryURL: registry.example.com
+
+imageRestrictedTags:
+       - latest
```

There should only be 2 spaces of indentation here
```suggestion
  - latest
```

---

Fixed